descriptors/ModuleDescriptor-template.json

```diff
@@ -138,7 +138,7 @@
     },
     {
       "id": "orders-storage.receiving-history",
-      "version": "3.1",
+      "version": "3.2",
```

Comment:
Shouldn't this version be bumped to `4.0` as well as we are bumping `pieces` to `4.0`?

Reply:
There is no POST or PUT endpoints for receiving-history. For GET endpoints, incompatible changes are changes that affect existing fields. Correct me if I'm wrong.